public/js/app.js

```diff
@@ -20,6 +20,92 @@ jQuery( document ).ready(function( $ ){
             });
         });
         $('.comment-btn').on('click', makeComment);
+
+        $('#followsLink, #followersLink').click(function(e) {
+            e.preventDefault();
+            var url = $(this).attr('data-url');
+            var title = /follows/.test(url) ? 'Following' : 'Followers';
+
+            $.getJSON(url, function(data) {
+                if (data.length === 0) {
+                    return;
+                }
+
+                $('#ajaxModal').on('show.bs.modal', function() {
+                    var modal = $(this);
+                    modal.find('.modal-title').text(title);
+
+                    var html = "", follow, display;
+
+                    data.forEach(function(user) {
+                        follow = user.checkFollow ? 'Following' : 'Follow';
+                        display = user.me ? "style='display:none'" : '';
+
+                        html += "<div class='row follows'>";
+                        html += "<div class='col-md-9'>";
+                        html += "<img align='left' class='img-circle img-responsive' src='";
+                        html += user.avatar + "' alt='Profile image' border-radius='100%''>";
+                        html += "<span><a href='/";
+                        html += user.username;
+                        html += "'>" + user.username + "</a></span>";
+                        html += "</div>";
+                        html += "<div class='col-md-3'>";
+                        html += "<button " + display + " data-id=" + user.id;
+                        html +" class='btn btn-primary follow'>" + follow + "</button>";
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Expected an assignment or function call and instead saw an expression.